quiz/src/main/java/com/epam/spring/homework3/quiz/controller/AnswerVariantController.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.spring.homework3.quiz.controller;
+
+import com.epam.spring.homework3.quiz.controller.dto.AnswerVariantDto;
+import com.epam.spring.homework3.quiz.controller.dto.group.OnCreate;
+import com.epam.spring.homework3.quiz.controller.mapper.AnswerVariantMapper;
+import com.epam.spring.homework3.quiz.service.AnswerVariantService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/answerVariant")
+@RequiredArgsConstructor
+@Api(tags = "AnswerVariantController description for SWAGGER documentation")
+public class AnswerVariantController {
```

Comment:
improve url format:

- most commonly used approach is with hyphens: change `answerVariant` to `answer-variant`
- same comment for url format as for `UserController`
- regarding `getAllAnswerVariantByParentQuestionId`, change url to:

`/question/{questionId}/answer-variant`